src/main/webapp/resources/js/pages/analysis/components/Analysis.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useContext } from "react";
+import { Tabs, Typography } from "antd";
+
+//import analysis components required by page
+import { AnalysisDetails } from "./AnalysisDetails";
+import { AnalysisSteps } from "./AnalysisSteps";
+import { AnalysisProvenance } from "./AnalysisProvenance";
+import { AnalysisOutputFiles } from "./AnalysisOutputFiles";
+import { AnalysisPhylogeneticTree } from "./AnalysisPhylogeneticTree";
+import { AnalysisBioHansel } from "./AnalysisBioHansel";
+import { AnalysisSistr } from "./AnalysisSistr";
+import { AnalysisError } from "./AnalysisError";
+import { getI18N } from "../../../utilities/i18n-utilties";
+import { SPACE_MD } from "../../../styles/spacing";
+
+import {
+  AnalysisContext,
+  AnalysisProvider
+} from "../../../contexts/AnalysisContext";
+import { AnalysisDetailsProvider } from "../../../contexts/AnalysisDetailsContext";
+
+const TabPane = Tabs.TabPane;
+const { Title } = Typography;
+
+const analysisTypesWithAdditionalPage = [
+  "BIO_HANSEL",
+  "SISTR_TYPING",
+  "PHYLOGENOMICS",
+  "MLST_MENTALIST"
+];
+
+export default function Analysis() {
+  const { analysisContext, analysisContextUpdateSubmissionName } = useContext(
+    AnalysisContext
+  );
+
+  return (
+    <>
+      <div
+        style={{
+          marginLeft: SPACE_MD,
+          marginRight: SPACE_MD,
+          marginTop: SPACE_MD
+        }}
+      >
+        <Title>{analysisContext.analysisName}</Title>
+        <AnalysisSteps />
+        <Tabs
+          defaultActiveKey={
+            analysisContext.isError ||
+            (analysisTypesWithAdditionalPage.indexOf(
+              analysisContext.analysisType.type
+            ) > -1 &&
+              analysisContext.isCompleted)
+              ? "0"
+              : "3"
+          }
+          animated={false}
+        >
+          {analysisContext.isCompleted ? (
+            [
+              analysisContext.analysisType.type === "BIO_HANSEL" ? (
+                <TabPane tab="bio_hansel" key="0">
+                  <AnalysisBioHansel />
```

Comment:
When you do these make sure you lazy load since they will not always be needed.

Reply:
Will do!